### DISCUSSION TITLE
Add README.md to link to Deployment Wiki

### DISCUSSION THREAD

**Discussion by hoangnguyen92dn** [nimblehq/flutter_templates#167] [RFCs]
## Issue

After generating a new project from `flutter_templates`, there are many fields in the deployment deployment that need to be replaced for Firebase App Distribution and TestFlight, It's too vague since there is no guideline inside the generated project, actually the Wiki for deployment doesn't link with configuration files so take time for newcomers to find out the instruction.

## Solution

- Add `README` for instructions that link to the Wiki: https://github.com/nimblehq/flutter_templates/wiki/Deployment#deployment-process

Sample:

https://github.com/nimblehq/android-templates/tree/develop/.cicdtemplate

## Who Benefits?

Developers

## What's Next?

- [ ] Address new issue to tackle this

**Comment by sleepylee** [comment]
> there are many fields in the deployment deployment that need to be replaced for Firebase App Distribution and TestFlight, It's too vague since there is no guideline inside the generated project

to be fair, the README is about the "template project" itself, while the WIKI could produce more details on how to use the "templated/generated project," which includes the concerns you mentioned. Indeed we could clarify the Wiki for people who are unfamiliar with the project, but the headline will be generic, users should read the WIKi and go through the headlines by themselves, the maintainers are not obligate to tell them "how to deploy" in the README.

**Reply by manh-t** [reply]
same thought as @sleepylee . 🙋

**Reply by luongvo** [reply]
@sleepylee @manh-t @hoangnguyen92dn We can add a new section called `Documentations` that links to our wiki page. We won't add any content in the README except the link.

**Reply by hoangnguyen92dn** [reply]
Yup. As I mentioned `Add README for instructions that link to the Wiki:`, means we don't need to have full content in the README, just need a headline to mention the deployment guideline in Wiki.

**Reply by sleepylee** [reply]
who will handle this 2-minute task?

**Reply by luongvo** [reply]
@sleepylee @hoangnguyen92dn This should be quick. I can take care of this update in [this task](https://github.com/nimblehq/flutter_templates/issues/172) as I need to cover the README content update, too. We don't need a separate ticket.

**Reply by luongvo** [reply]
Added in https://github.com/nimblehq/flutter-templates/pull/174 ✅
